docs/src/reference/models.md

```diff
@@ -40,16 +40,16 @@ set_string_names_on_creation
 ```@docs
 set_optimizer
 optimizer_with_attributes
-get_optimizer_attribute
-set_optimizer_attribute
-set_optimizer_attributes
 get_attribute
 set_attribute
 set_silent
 unset_silent
 set_time_limit_sec
 unset_time_limit_sec
 time_limit_sec
+get_optimizer_attribute
+set_optimizer_attribute
+set_optimizer_attributes
 ```
```

Comment:
maybe even:

```suggestion
` ` `

### Legacy
` ` `@docs
get_optimizer_attribute
set_optimizer_attribute
set_optimizer_attributes
` ` `
```
(added the space to that preview compiles properly...)

Reply:
I added a `!!! compat` warning to the docstrings. I don't know if we need to make an explicit `Legacy` section.